src/redux/actions.ts

```diff
@@ -88,6 +93,39 @@ const syncFearConditioningModule = async (
   }
 }
 
+const syncCriterionModule = async (
+  experiment: ExperimentCache,
+  mod: ExperimentModule<CriteriaModuleState>,
+  onModuleSync: ModuleSyncCallback,
+) => {
+  // Perform POST request for each recordered question
+  try {
+    await Promise.all(
+      mod.moduleState.questions.map(async (question, index) => {
+        // Don't attempt sync if no response
+        if (question.value === undefined) return
```

Comment:
No response is modeled in the database. You can submit `null` to answer. I think that's probably more appropriate to explicitly say that they've skipped the question.

Reply:
Ah yes good point!